translations/base-ja.yaml

```
@@ -30,32 +30,32 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        shapez.ioは無限のマップ内で様々な"形"を資源とし、段々と複雑になっていく形の作成や合成の自動化を目指して工場を構築するゲームです。
+        求められた"形"を届けることで様々なアップグレードを手に入れることができ、さらに工場の効率を上げることができます。
 
-        As the demand for shapes increases, you will have to scale up your factory to meet the demand - Don't forget about resources though, you will have to expand across the [b]infinite map[/b]!
+        "形"の需要が増えるにつれ、工場を拡大する必要があります - より多くの資源を手に入れるために[b]無限のマップ[!b]を開拓していく必要があります。
 
-        Soon you will have to mix colors and paint your shapes with them - Combine red, green and blue color resources to produce different colors and paint shapes with it to satisfy the demand.
+        "形"に色を塗る必要もあります - 赤、緑、青を混ぜ、さまざまな色を作り需要に応じて色が塗られた"形"を製造できます。
 
-        This game features 18 progressive levels (Which should keep you busy for hours already!) but I'm constantly adding new content - There is a lot planned!
+        現在このゲームには18個のレベルがあり、今後これを拡大していく予定です - 他にも様々なアップデートを準備しています!
 
-        Purchasing the game gives you access to the standalone version which has additional features and you'll also receive access to newly developed features.
+        このゲームを買うことによってスタンドアローン版の様々な機能が使えるようになり、新しいアップデートもすぐに手に入ります。
 
-        [b]Standalone Advantages[/b]
+        [b]スタンドアローン版で使える機能[/b]
 
         [list]
-            [*] Dark Mode
-            [*] Unlimited Waypoints
-            [*] Unlimited Savegames
-            [*] Additional settings
-            [*] Coming soon: Wires & Energy! Aiming for (roughly) end of July 2020.
-            [*] Coming soon: More Levels
-            [*] Allows me to further develop shapez.io ❤️
+            [*] ダークモード
+            [*] ウェイポイントの制限がありません
+            [*] セーブの数の制限がありません
+            [*] 様々な設定
+            [*] 追加予定: ワイヤーとエネルギー! (7月末を目指しています)
+            [*] 追加予定: より多くのレベル
+            [*] shapez.io開発が助けられます❤️
         [/list]
 
-        [b]Future Updates[/b]
+        [b]今後の予定[/b]
 
-        I am updating the game very often and trying to push an update at least every week!
+       　週に一回はアップデートを出せることを目指しています。
```

Indent error. A wide space is used instead of ASCII space.
```suggestion
        週に一回はアップデートを出せることを目指しています。
```